docs/api/products-api.md

```diff
@@ -0,0 +1,80 @@
+## 1.GET /api/products/all
+<a name="api-products-get" hidden></a>
+
+| Route                 | Description                        | Access      |
+|:----------------------|:-----------------------------------|:------------|
+| GET /api/products/all | UI Get All Products                | Public      |       
+
+## Params
+1. _id
+    - type: string
+    - required: true
+    - regexp: mongoose_id
+2. name
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 100
+        2. startWith: [a-z] || [A-Z]
+3. type
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 10
+        2. startWith: [a-z] || [A-Z]
+        3. oneOf: ['food','drinks']
+4. image
+    - type: string
+    - required: optional
+    - regexp: length: min: 1 - max: 1000
+5. price
+    - type: double
```

Comment:
**Modifier format**
MongoDb does not support *double* type, do you mean *Number*?

Reply:
omg I dunt remember :((